assets/scss/imageSlider.scss

```
@@ -0,0 +1,50 @@
+.imageSlider{
```

```suggestion
.imageSlider {
```
Ima jos ovih gresaka ispod